modules/programs/rofi.nix

```diff
@@ -195,6 +195,24 @@ in
       description = "Whether to run rofi fullscreen.";
     };
 
+    location = mkOption {
+      default = null;
+      type = types.nullOr types.int;
+      description = "Location on screen (1-8)";
```

Comment:
Could have `type = types.nullOr (types.enum [1 2 3 4 5 6 7 8]);` here.

Reply:
Hmm, actually I think it in this case is fair to simply set the default to 0 and include the pattern in the description. Something like

```nix
    location = mkOption {
      default = 0;
      type = types.enum (range 0 8);
      description = ''
        Location on screen following the pattern

        <informaltable frame="none"><tgroup cols="1"><tbody>
        <row><entry>1</entry><entry>2</entry><entry>3</entry></row>
        <row><entry>8</entry><entry>0</entry><entry>4</entry></row>
        <row><entry>7</entry><entry>6</entry><entry>5</entry></row>
        </tbody></tgroup></informaltable>
      '';
    };
```